submit-web/src/components/SubmissionItem/ContactInformation/ContactInformationEntityView.tsx

```diff
@@ -104,7 +180,52 @@ export const ContactInformationEntityView = () => {
     defaultValues,
   });
 
-  const { handleSubmit } = methods;
+  const { handleSubmit, watch, clearErrors, setValue } = methods;
+
+  const primaryContactUserId = watch("primaryContact.accountUserId");
+  const secondaryContactUserId = watch("secondaryContact.accountUserId");
+  const isPrimaryUserSelected = Boolean(primaryContactUserId);
+  const isSecondaryUserSelected = Boolean(secondaryContactUserId);
+
+  useEffect(() => {
+    if (isPrimaryUserSelected) {
+      clearErrors([
+        "primaryContact.givenName",
+        "primaryContact.surname",
+        "primaryContact.position",
+        "primaryContact.company",
+        "primaryContact.workPhoneNumber",
+        "primaryContact.workEmailAddress",
+      ]);
+      setValue("primaryContact.givenName", "");
```

Comment:
Just heads up from the task:
"The Drop-down has a list of all the active users for this project. **Selecting a user populate the fields in the form**."